install_config/aggregate_logging.adoc

```diff
@@ -313,6 +330,10 @@ The default is a generated key.
 |The location of the CA cert Elasticsearch uses for the external TLS
 server cert. The default is the internal CA.
 
+|`openshift_logging_fluentd_image`
+|The image version for Fluentd.
+For example: `openshift_logging_fluentd_image=<registry_url>:5000/openshift3/logging-fluentd:v3.X.X`
```

Comment:
Shall we use offical images as an example? registry.redhat.io/openshift3/ose-logging-fluentd:v3.11